training/provision/playbook.yml

```diff
@@ -15,16 +15,41 @@
   - name: Gather facts for first time
     ansible.builtin.setup:
 
+  - name: Get size of root filesystem
+    ansible.builtin.shell:
+      cmd: |
+        df -h /
+        df -h /var/tmp/
+        podman system info
+
   - name: Check Podman Present
     ansible.builtin.package:
       name: podman
       state: present
 
+  - name: Login to default registry and create ${XDG_RUNTIME_DIR}/containers/auth.json
+    containers.podman.podman_login:
+      username: "{{ registry_user }}"
+      password: "{{ registry_password }}"
+      registry: quay.io
+
+  - name: Building an image with ssh key
```

Comment:
Wait...unless I am misunderstanding the ansible abstractions here, this isn't doing a build at all.  This is just running some commands in the container and throwing away the result.

Reply:
IOW, the more desired flow here would be to build a derived image, push it to a registry (so the host can do in-place updates, if we care to test that) and then also use the bootc install flow.

Alternatively, it works to do a local build for testing and install that, but it obviously disables in-place updates, in case we care about that.

Reply:
Thanks @cgwalters for catching it.  The module was replaced with the correct one now.
Regarding the in-place updates, that is definitely needed. Lets file a GH issue in this repo and it will be handled in a followup PR.